.ado/windows-vs-pr.yml

```diff
@@ -90,6 +90,7 @@ jobs:
           testSelector: testAssemblies
           testAssemblyVer2: |
             Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.exe
+            Microsoft.ReactNative.IntegrationTests/Microsoft.ReactNative.IntegrationTests.exe
```

Comment:
Should we also add Microsoft.ReactNative.ComponentTests to this?

Reply:
I did it initially, but the ComponentTests do not compile for x86 yet: stdcall vs cdecl issue. We need to resolve this issue first before we can enable it.